framework/src/media/utils/MediaUtils.cpp

```diff
@@ -68,10 +72,227 @@ audio_type_t getAudioTypeFromPath(std::string datapath)
 	} else if ((extension.compare("") == 0) || (extension.compare("pcm") == 0) || (extension.compare("raw") == 0)) {
 		medvdbg("audio type : pcm\n");
 		return AUDIO_TYPE_PCM;
+	} else if (extension.compare("wav")) {
+		medvdbg("audio type : wav\n");
+ 		return AUDIO_TYPE_WAVE;
 	} else {
 		medvdbg("audio type : unknown\n");
 		return AUDIO_TYPE_INVALID;
 	}
 }
+
+void mp3_header_parsing(unsigned char * header, unsigned int *channel, unsigned int *sample_rate, audio_format_type_t *pcmFormat)
+{
+	*pcmFormat = AUDIO_FORMAT_TYPE_S16_LE;
+	
+	unsigned char bit;
+	unsigned int MPEG_Version;	
+	bit = header[1];
+	bit >>= 3;
+	bit &= (unsigned char)3;
+
+	if (bit == 0) {
+		MPEG_Version = 2;
+	}
+	else if (bit == 2) {
+		MPEG_Version = 1;
+	}
+	else if (bit == 3) {
+		MPEG_Version = 0;
+	}
+	bit = header[2];
+	bit >>= 2;
+	bit &= 3;
+	
+	if (bit == 0) {
```

Comment:
1. apply coding rule
2. In this case, swith-case is much better options because you marked else state as a reserved.